iidm/iidm-api/src/test/java/com/powsybl/iidm/network/TestImporter.java

```diff
@@ -56,4 +56,19 @@ public Network importData(ReadOnlyDataSource dataSource, NetworkFactory networkF
         }
         return networkFactory.createNetwork("mock", "test");
     }
+
+    @Override
+    public void update(Network network, ReadOnlyDataSource dataSource, Properties parameters, ReportNode reportNode) {
+        if (reportNode != null) {
+            reportNode.newReportNode()
+                    .withMessageTemplate("test", "Update model ${model}")
```

Comment:
Can you migrate this message to the new ReportNode v3 API?
- replace this line with
  - `.withMessageTemplate("testUpdateModel")`
- add the following line in `iidm/iidm-api/src/test/resources/i18n/reports.properties`:
  - `testUpdateModel = Update model ${model}`

Reply:
Done. I've changed the call to `withMessageTemplate` to avoid a deprecation warning:

```java
reportNode.newReportNode()
        .withResourceBundles(PowsyblCoreTestReportResourceBundle.TEST_BASE_NAME)
        .withMessageTemplate("testUpdateModel")
        .withUntypedValue("model", "eurostagTutorialExample1")
        .add();
```

Reply:
Thanks!